core/src/main/java/org/pentaho/agilebi/modeler/nodes/AbstractAvailableItem.java

```diff
@@ -79,4 +81,41 @@ public String getClassname() {
   public void setClassname( String classname ) {
     this.classname = classname;
   }
+
+  @Bindable
+  public String getAltText() {
+    return this.altText;
+  }
+
+  @Bindable
+  public void setAltText( String altText ) {
+    this.altText = altText;
+  }
+
+  /**
```

Comment:
helper function to avoid DRY for grabbing value from property file and calling #setAltText(). similar code in AbstractMetaDataModelNode.java